packages/styles/src/components/button.scss

```diff
@@ -127,6 +123,20 @@
   }
 }
 
+.btn-default,
+.btn-secondary,
+.btn-link,
+.btn-tertiary {
```

Comment:
Here you could create a `transparent-background-button` mixin that you will use in secondary and tertiary button code blocks. This would help keep the code separated and therefore easy to read.

Reply:
Good idea, and thanks for naming. I used a placeholder to avoid duplicating output.